wiki/People/Global_Moderation_Team/en.md

```diff
@@ -135,6 +135,7 @@ The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all o
 | ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) | French, Spanish | Chat moderation |
 | ::{ flag=ID }:: [Shurelia](https://osu.ppy.sh/users/3807986) | Indonesian | Chat moderation |
 | ::{ flag=ID }:: [Sies](https://osu.ppy.sh/users/6491991) | Indonesian, some Vietnamese | Chat moderation, forum moderation |
+| ::{ flag=AT }:: [Stefan](https://osu.ppy.sh/users/626907) | German | Forum moderation |
```

Comment:
I think he also speaks Serbian iirc

Reply:
Let me ask him real quick if it's true.

Reply:
Oh okay now i remember when i asked him once before, he's actually Serbian.

Reply:
Internally he isn't listed for that language, just OnosakiHito and 0x84f come to mind. Don't think he's moderating it?

Reply:
Not sure, but all i know that he is also Serbian, didn't ask if he also speaks Serbian, probably it should be obvious? I don't really know...

Reply:
He hath become Serbian